openstack/designate/templates/etc/_secrets.conf.tpl

```diff
@@ -1,9 +1,9 @@
 [DEFAULT]
-transport_url = rabbit://{{ .Values.rabbitmq.users.default.user | default "rabbitmq" }}:{{ .Values.rabbitmq.users.default.password }}@{{ include "rabbitmq_host" . }}:{{ .Values.rabbitmq.port | default 5672 }}/
+transport_url = rabbit://{{ .Values.rabbitmq.users.default.user | default "rabbitmq" | include "resolve_secret_urlquery" }}:{{ .Values.rabbitmq.users.default.password | include "resolve_secret_urlquery" }}@{{ include "rabbitmq_host" . }}:{{ .Values.rabbitmq.port | default 5672 }}/
 
 [keystone_authtoken]
 username = {{ .Values.global.designate_service_user | default "designate" }}
-password = {{ .Values.global.designate_service_password }}
+password = {{ .Values.global.designate_service_password | include "resolve_secret" }}
 
 [storage:sqlalchemy]
```

Comment:
Unfortunately, resolve_secret hasn't been added for secrets in db_url_pxc in utils helm chart...